docs/content/Coding Milestones/PoC2023q1/mailbox-controller.md

```diff
@@ -44,31 +44,40 @@ APIExport in those workspaces; this should be a client config that is
 able to read/write in all clusters.  For example, that is in the
 kubeconfig context named `base` in the kubeconfig created by `kcp
 start`.  Finally, the controller also needs a kube client config that
-is pointed at the edge service provider workspace and is authorized to
-consume the `Workspace` objects from there.
+is pointed at the root workspace and is authorized to consume the `Workspace`
+objects from there.
 
 The command line flags, beyond the basics, are as follows.
 
 ``` { .bash .no-copy }
+      --add_dir_header                   If true, adds the file directory to the header of the log messages
```

Comment:
Please note the qualifier above: "beyond the basics".  It was my intent to not repeat the standard command line flags because (a) they add a lot of noise and (b) they may change without our control or notice.

Reply:
Changed this to add/remove only the flags from this PR